docs/pom.xml

```diff
@@ -73,6 +74,11 @@
                                     <include>attributes.adoc</include>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${project.basedir}/../target/asciidoc/generated/config/</directory>
+                                    <include>quarkus-roq-data.adoc</include>
```

Comment:
You can move this include to the `<resource>` above. See https://github.com/quarkiverse/quarkus-kerberos/blob/main/docs/pom.xml#L69 as an example